plugin/src/main/java/org/eobjects/build/AbstractDotnetTestMojo.java

```diff
@@ -3,20 +3,24 @@
 import java.io.File;
 
 import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Parameter;
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 
 public abstract class AbstractDotnetTestMojo extends AbstractDotnetMojo {
 
+	@Parameter(property = "dotnet.test.outputxml", required = false, defaultValue = "TestResults.xml")
```

Comment:
Two small remarks:
(1) The indentation is inconsistent, should be (4) spaces
(2) Shouldn't we put TestResults.xml in some folder like `target/` or `bin/` or `obj/` by default? Usually people will have added those folders to .gitignore or whatever else is controlling SCM ignore directives. Usually people don't want a build to add anything to SCM since many CI environment will consider that a build failure if SCM has changes after build